src/history.h

```diff
@@ -227,7 +227,7 @@ class history_t {
 
     // Searches history.
     bool search(history_search_type_t search_type, wcstring_list_t search_args,
-                const wchar_t *show_time_format, io_streams_t &streams);
+                const wchar_t *show_time_format, long max_count, io_streams_t &streams);
```

Comment:
There is currently only one caller of this function: `builtin_history`. It defaults this parameter to LONG_MAX. I suppose we could make that the default for this method but then why not do the same thing for `show_time_format`? And why bother to do either if, as seems likely, we won't be adding new callers of this method?